src/account/Template.tsx

```diff
@@ -83,6 +84,26 @@ export default function Template(props: TemplateProps<KcContext, I18n>) {
                 ]}
             />
 
+            {(kcContext.properties?.DSFR_NOTICE_TITLE || kcContext.properties?.DSFR_NOTICE_DESCRIPTION) && (
```

Comment:
Why the optional chaining `?` we know `kcContext.properties` is always defined here.